docs/docs/using-semaphore/tests/markdown-reports.md

```diff
@@ -0,0 +1,322 @@
+---
+description: Create customized reports
+sidebar_position: 1
+---
+
+# Markdown Reports
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Available from '@site/src/components/Available';
+import VideoTutorial from '@site/src/components/VideoTutorial';
+import Steps from '@site/src/components/Steps';
+
+Create rich custom reports and view them right next to your workflows and jobs.
+
+## Overview {#overview}
+
+The new Reports tab makes it easy to add clear, custom markdown reports right inside your job and workflow pages. You can include charts, summaries, and extra details without leaving Semaphore, so sharing build results and pipeline information is simpler.
+
+Markdown Reports feature:
+
+- **Fully rendered Markdown**: view Markdown files directly in the workflow or job pages, which are fully formatted for easy reading.
+- **Mermaid.js support**: create live charts and diagrams to show your pipeline steps with [Mermaid.js](https://mermaid.js.org/)
+- **Quick report generation** : use the [test-results](../../reference/toolbox#test-results) CLI to make basic job reports in seconds
+- **Fully customizable**: include test results, deployment notes, or any other information you need to organize and explain your CI/CD process
+
+For example, you can use them to show information about your build.
+
+![Build info report](./img/build-info.jpg)
+
+Visually show how long each command in your job takes
+
+![Gantt commands](./img/timeline.jpg)
+
+Obtain resource utilization metrics.
+
+![Utilization metrics](./img/resource-metrics.jpg)
+
+You can even use images in your reports.
+
+![Embedded Image](./img/markdown-reports-dependencies-example.jpg)
```

Comment:
Awesome page 🙌 

I only found this part confusing because this image looks a lot like it's `mermaid.js` chart and not just an example of a random image you can embed.

I would consider: 
1. Using some other image, because that one looks a lot like it's actually markdown and not PNG. Maybe a cat image 🐱 Do kids these days still use those as a placeholder images or it's overused trope?
2. Would be cool if we had an actual example of generating this with dependency cruiser in markdown, or at least as an image that gets uploaded to report.